src/test/java/ch/jalu/collectionbehavior/verification/MapModificationVerifier.java

```diff
@@ -0,0 +1,107 @@
+package ch.jalu.collectionbehavior.verification;
+
+import ch.jalu.collectionbehavior.model.MapMethod;
+import ch.jalu.collectionbehavior.model.ModificationBehavior;
+import ch.jalu.collectionbehavior.model.SetMethod;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import static ch.jalu.collectionbehavior.model.MapCreator.A_VALUE;
+import static ch.jalu.collectionbehavior.model.MapCreator.B_VALUE;
+import static ch.jalu.collectionbehavior.model.MapCreator.D_VALUE;
+
+public final class MapModificationVerifier {
+
+    private MapModificationVerifier() {
+    }
+
+    public static void testMethods(Map<String, Integer> originalMap, ModificationBehavior expectedBehavior) {
```

Comment:
Check that the most constellations possible have been tested in this class. Maybe also for set&list